c7n/resources/route53.py

```diff
@@ -208,6 +208,65 @@ def process_resource_set(self, client, domains, keys):
                 TagsToDelete=keys)
 
 
+@ResourceRecordSet.action_registry.register('remove-recordset')
+class ResourceRecordSetRemove(BaseAction):
+    """Action to delete resource records from Route 53 hosted zones.
+
+    It is recommended to use a filter to avoid unwanted deletion
+    of R53 records from all hosted zones.
+
+    :example:
+
+    .. code-block:: yaml
+
+            policies:
+              - name: route53-remove-filtered-records
+                resource: aws.rrset
+                filters:
+                  - type: value
+                    key: AliasTarget.DNSName
+                    value: "email.gc.example.com."
+                actions:
+                  - type: remove-recordset
+
+    """
+    schema = type_schema('remove-recordset',)
+    permissions = ('route53:ChangeResourceRecordSets',)
+    keys = (
+        'Name', 'Type', 'TTL', 'SetIdentifier', 'Region', 'AliasTarget', 'ResourceRecords')
+
+    def process(self, recordsets):
+        client = local_session(self.manager.session_factory).client('route53')
+        try:
+            for rrset in recordsets:
+                if rrset['Type'] in ('NS', 'SOA'):
```

Comment:
this could use a comment re why their being exempted. we have a filter_resources element that does attribute based filtering and logging of the resources when certain resources aren't supported by an action.

Reply:
Updated the comment on why those two records are exempted.